Fatal errors when building the Dockerfile

I just cloned the repo and then built the Dockerfile with the command

```
docker build -t mosca .
```

Two fatal errors were thrown. One was:

```
../lib/kerberos.h:5:27: fatal error: gssapi/gssapi.h: No such file or directory
```

which I was able to solve by adding `apk add krb5-dev` to the Dockerfile.

The other error was:

```
make: Entering directory '/usr/src/app/node_modules/ascoltatori/node_modules/zmq/build'
  CXX(target) Release/obj.target/zmq/binding.o
../binding.cc:28:17: fatal error: zmq.h: No such file or directory
```

I haven't had any luck solving this second error. The install continues to run to completion after these errors. Will these errors impact Mosca (specifically as a Kafka-MQTT bridge)? Is there a way to solve this second error?

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,6 +1,6 @@
 # Mosca
 #
-# VERSION 0.2.1
+# VERSION 2.5.2
 
 FROM mhart/alpine-node:4
 MAINTAINER Matteo Collina <hello@matteocollina.com>
@@ -11,7 +11,7 @@ WORKDIR /usr/src/app/
 COPY ./ /usr/src/app/
 
 RUN apk update && \
-    apk add make gcc g++ python git && \
+    apk add make gcc g++ python git zeromq-dev krb5-dev && \
     npm install --unsafe-perm --production && \
     apk del make gcc g++ python git
 

--- a/Dockerfile-rpi
+++ b/Dockerfile-rpi
@@ -1,6 +1,6 @@
 # Mosca
 #
-# VERSION 0.3.0
+# VERSION 2.5.2
 
 FROM hypriot/rpi-node:6
 MAINTAINER Matteo Collina <hello@matteocollina.com>
@@ -11,9 +11,10 @@ WORKDIR /usr/src/app/
 COPY ./ /usr/src/app/
 
 RUN apt-get update && \
-    apt-get install make gcc g++ python git && \
+    apt-get install make gcc g++ python git libzmq3-dev libkrb5-dev && \
     npm install --unsafe-perm --production && \
-    apt-get remove make gcc g++ python git
+    apt-get remove make gcc g++ python git && \
+    apt-get autoremove
 
 EXPOSE 80
 EXPOSE 1883

--- a/Dockerfile-rpi-hub
+++ b/Dockerfile-rpi-hub
@@ -1,6 +1,6 @@
 # Mosca
 #
-# VERSION 0.2.1
+# VERSION 2.5.2
 
 FROM ma314smith/rpi2-node-qemu:4
 MAINTAINER Matteo Collina <hello@matteocollina.com>
@@ -13,9 +13,10 @@ WORKDIR /usr/src/app/
 COPY ./ /usr/src/app/
 
 RUN apt-get update && \
-    apt-get install make gcc g++ python git && \
+    apt-get install make gcc g++ python git libzmq3-dev libkrb5-dev && \
     npm install --unsafe-perm --production && \
-    apt-get remove make gcc g++ python git
+    apt-get remove make gcc g++ python git && \
+    apt-get autoremove
 
 EXPOSE 80
 EXPOSE 1883

--- a/examples/kafka/Dockerfile
+++ b/examples/kafka/Dockerfile
@@ -2,7 +2,7 @@ FROM mhart/alpine-node:4
 MAINTAINER Robert Fuller <fullergalway@gmail.com>
 
 RUN apk update && \
-    apk add make gcc g++ python git 
+    apk add make gcc g++ python git zeromq-dev krb5-dev
 
 RUN mkdir -p /usr/src/app
 RUN mkdir -p /app/db

